Files/ViewModels/SettingsViewModel.cs

```diff
@@ -515,7 +515,7 @@ public bool PinRecycleBinToSideBar
         /// </summary>
         public bool IsAcrylicDisabled
```

Comment:
That's a weird setting name - `IsAcrylicDisabled`. Wouldn't it be better if it was called `IsAcrylicEnabled`?

Reply:
@d2dyno1 This was added before we started using converters, we can definitely change around the name. Do you want to work on this?

Reply:
@yaichenbaum sure!